rust/src/mysql/logger.rs

```diff
@@ -0,0 +1,117 @@
+/* Copyright (C) 2022 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+// written by linqiankai <linqiankai@geweian.com>
+//
+use crate::jsonbuilder::{JsonBuilder, JsonError};
+use crate::mysql::mysql::*;
+
+fn log_mysql(tx: &MysqlTransaction, _flags: u32, js: &mut JsonBuilder) -> Result<(), JsonError> {
+    js.open_object("mysql")?;
+    js.set_uint("tx_id", tx.tx_id)?;
```

Comment:
Is it meaningful to log the tx id ?

Reply:
No, all of fields are  based on my project, if you have any idea in logger, please comment, thx.

Reply:
Fixed, please check again.